autoscheduler/frontend/src/components/MeetingCard.tsx

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable react/prop-types */
```

Comment:
Can we remove this line, per your change in `.eslintrc.js`

Reply:
Yes